### PR TITLE
adding 10x wrapper function

### PR DESCRIPTION
**Pull request by ekiernan**
This PR adds a new docker to the ATAC workflow with SnapATAC2 v2.7.0. It modifies the existing CreateFragmentFile script to use the new SnapATAC2 10 wrapper function that produces library-level metrics, a fragment file, and an h5ad metric file in one function. We added the new library level metrics as a CSV output of the workflow, as well as an out of the Multiome and Paired-tag workflows.

### CODE REVIEW

File: pipelines/skylab/atac/atac.wdl [hunk 1]
```diff
@@ -58,7 +58,7 @@ workflow ATAC {
   String cutadapt_docker = "cutadapt:1.0.0-4.4-1686752919"
   String samtools_docker = "samtools-dist-bwa:3.0.0"
   String upstools_docker = "upstools:1.0.0-2023.03.03-1704300311"
-  String snap_atac_docker = "snapatac2:1.0.9-2.6.3-1715865353"
+  String snap_atac_docker = "snapatac2:lk-PD-2738"
```
Comment (aawdeh) [inline pipelines/skylab/atac/atac.wdl:61]: Can we change the branch name in the docker path to the docker version?

Comment (ekiernan) [reply pipelines/skylab/atac/atac.wdl]: thank you! Just added the updated version

**Review by nikellepetrillo — APPROVED** [review]
looks good to me!

**Comment by ekiernan** [comment]
Multiome Scientific: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/83618/

Paired-tag scientific: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/83619/

**Review by aawdeh — APPROVED** [review]
LGTM

**Pull request merged**